### ISSUE TITLE
Importing _self doesn't work

### ISSUE THREAD

**Issue by janoszen** [jtwig/jtwig#333]
In Twig the current template can be imported using `_self` for the purposes of using a macro:

```twig
{% import _self as m %}

{% macro test() %}{% spaceless %}
    Hello world!
{% endspaceless %}{% endmacro %}

<div>
{{ m.test() }}
</div>
```

However, in jTwig strict mode this causes an error:

```
Exception in thread "main" org.jtwig.exceptions.ResolveValueException: classpath:example.twig (Line: 1, Column: 11) -> Variable '_self' undefined
	at org.jtwig.render.expression.calculator.VariableExpressionCalculator.calculate(VariableExpressionCalculator.java:16)
	at org.jtwig.render.expression.calculator.VariableExpressionCalculator.calculate(VariableExpressionCalculator.java:9)
	at org.jtwig.render.expression.CalculateExpressionService.calculate(CalculateExpressionService.java:14)
	at org.jtwig.render.node.renderer.ImportNodeRender.render(ImportNodeRender.java:32)
	at org.jtwig.render.node.renderer.ImportNodeRender.render(ImportNodeRender.java:20)
	at org.jtwig.render.node.RenderNodeService.render(RenderNodeService.java:21)
	at org.jtwig.render.node.renderer.CompositeNodeRender.render(CompositeNodeRender.java:21)
	at org.jtwig.render.node.renderer.CompositeNodeRender.render(CompositeNodeRender.java:13)
	at org.jtwig.render.node.RenderNodeService.render(RenderNodeService.java:21)
	at org.jtwig.JtwigTemplate.render(JtwigTemplate.java:97)
	at org.jtwig.JtwigTemplate.render(JtwigTemplate.java:79)
	at SimpleApp.main(SimpleApp.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
```

**Comment by lyncodev** [comment]
Should be fixed asap.

**Comment by lyncodev** [comment]
Fixed in release 5.85.1.RELEASE
